content/concern/dump.md

```diff
@@ -6,10 +6,142 @@ categories: [concerns]
 keywords: []
 aliases: []
 toc: false
-draft: false
+draft: true
 ---
 
 This is a handler that dumps the entire request and response into a log file. It
 should only be used in development mode for debugging purpose as it is very slow.
 
 
+### Introduction 
+
+This handler can log HTTP request/response info based on configuration.
+
+This is a generic dump handler may be useful to troubleshooting in developing/testing environment.
+
+This handler is not suggested to be used on production because it will impact the overall performance.
+
+The default request/response log will be dump.log config.
```

Comment:
Or whatever's configured in logback.xml right?

Reply:
Yes. It won't be automatically generated in logback.xml, so there is not "default" thing. User needs to add it manually. BTW, do I need to add dump.log in logback.xml when doing the code generation like "audit.log"?

Reply:
@BalloonWen Good point, i hadn't considered that. Yeah i think codegen should include the dump.log reference. @stevehu what do you think?